main.go

```diff
@@ -289,6 +289,25 @@ func visit(url *url.URL) {
 		log.Fatalf("failed to read response: %v", err)
 	}
 
+	// Print SSL/TLS version which is used for connection
+	connectedVia := "plaintext"
+	if resp.TLS != nil {
+		switch resp.TLS.Version {
+		//lint:ignore SA1019
+		case tls.VersionSSL30:
```

Comment:
try 

case tls.VersionSSL30: //lint:ignore SA1019 

It might require a comment after the ignore SA1019 but I cannot think of one. Honestly I'm more tempted to adjust the TLSConfig settings to never negotiate this setting so we don't have to check for it.

Reply:
Done. Please retest

Reply:
@dpanic if your last change doesn't work, maybe try the syntax shown here:

https://stackoverflow.com/a/66291436/903870

Reply:
Ok thank you, waiting for you guys to check CI/CD

Reply:
Removed SSLv3. 
Added TSLv1.0 as minimum version for connection.

Reply:
@davecheney probably you missed my commits. Please take look at it now

Reply:
@atc0005 this is done. Please review

Reply:
@dpanic I'm an observer of the project (actively learning/improving Go skills) and not directly associated with it. I thought I saw a potential opportunity to assist and spoke up. Apologies if I gave an impression otherwise.